book/website/collaboration/hybrid-collab/hybrid-collab-guidelines.md

```diff
@@ -11,60 +11,68 @@
 ### Scheduling
 
 - Organising a hybrid event is like organising two events: in-person and online. 
-    - If you're separating in-person and online (a couple of days in-person and a couple of days online), it might be easier to handle.
-    - If both modes take place simultaneously it takes different planning and it is more challenging as you will be running two events at the same time. 
+    - It might be easier to handle if you're separating in-person and online (a couple of days in-person and a couple of days online).
+    - If both modes take place simultaneously, it takes different planning, and it is more challenging as you will be running two events simultaneously. 
 
-- Online participants might have to join at times that are not core working hours for them. Keep this in mind, especially for interactive parts.
-- If some aspects of the agenda only apply to one participant group, be clear about this. For example, do not schedule a networking session that turns out to just be the in-person coffee break without being clear how you expect online participants to network. 
+- Online participants might have to join at times that are not core working hours. Keep this in mind, especially for interactive parts.
+- If some aspects of the agenda only apply to one participant group, be clear about this. For example, do not schedule a networking session that turns out to be an in-person coffee break without being clear about how you expect online participants to network.
+- Organize multiple sessions or select strategic times to accommodate online participants from different timezones to address possible scheduling challenges.
 
 ### Location
 
-- Reserve rooms for break out interaction with online participants.
-- For in-person attendees, it might take some time to physically move to other rooms, whereas for people online it might be faster to move to a different breakout room. Please consider such time requirements when switching rooms either physically or virtually. 
-- Make sure technical set up is in order: online audience should be able to hear the speaker and questions asked and be able to see the slides. 
+- Reserve rooms for break-out interaction with online participants.
+- For in-person attendees, it might take some time to physically move to other rooms physically, whereas for people online, it might be faster to move to a different breakout room. Please consider such time requirements when switching rooms either physically or virtually. 
```

Comment:
```suggestion
- For in-person attendees, it might take some time to physically move to other rooms physically, whereas for people online, it might be faster to move to a different breakout room.
Please consider such time requirements when switching rooms either physically or virtually. 
```